python/lsst/meas/algorithms/brightStarStamps.py

```diff
@@ -254,9 +261,15 @@ def initAndNormalize(cls, starStamps, innerRadius, outerRadius,
         bss._checkNormalization(True, innerRadius, outerRadius)
         bss._innerRadius, bss._outerRadius = innerRadius, outerRadius
         # Apply normalization
-        for stamp in bss._stamps:
-            stamp.measureAndNormalize(annulus, statsControl=statsControl, statsFlag=statsFlag,
-                                      badMaskPlanes=badMaskPlanes)
+        for j, stamp in enumerate(bss._stamps):
+            try:
+                stamp.measureAndNormalize(annulus, statsControl=statsControl, statsFlag=statsFlag,
+                                          badMaskPlanes=badMaskPlanes)
+            except ValueError:
+                if discardNanFluxObjects:
+                    bss._stamps.pop(j)
+                else:
+                    stamp.annularFlux = np.nan
```

Comment:
It's not obvious to me what use this would serve, so it may be worth explaining here or in the function docstring under Notes.

Reply:
Which aspect are you referring to in particular: the `if/else`, the `pop`, setting the `stamp.annularFlux` to `np.nan`, or something else? Happy to add a brief comment into the text to clarify.

Reply:
The setting to nan part. A brief comment on why you might want to keep the stamp anyway would be helpful (my only immediate guess would be for bookkeeping so that you know that it tried and failed to measure something).

Reply:
Thanks Dan. Added the following comment block into the file:
```
# Optionally keep NaN flux objects, for bookkeeping purposes,
# and to avoid having to re-find and redo the preprocessing
# steps needed before bright stars can be subtracted.
```